tests/test_utils.py

```diff
@@ -1,4 +1,26 @@
+from binascii import unhexlify
+
 # Default variables used for the ntlm_context
 username = 'username'
 domain = 'domain'
 password = 'password'
+
+timestamp = unhexlify('0000000000000000')
+client_challenge = unhexlify('aaaaaaaaaaaaaaaa')
+session_base_key = unhexlify('55555555555555555555555555555555')
```

Comment:
Is there any reason to run these through `unhexlify` instead of just using bytestrings directly?